package.json

```diff
@@ -37,7 +37,7 @@
     "download:kubectl": "yarn run ts-node build/download_kubectl.ts",
     "download:helm": "yarn run ts-node build/download_helm.ts",
     "build:tray-icons": "yarn run ts-node build/build_tray_icon.ts",
-    "lint": "yarn run eslint --ext js,ts,tsx --max-warnings=0 .",
+    "lint": "PROD=true yarn run eslint --ext js,ts,tsx --max-warnings=0 .",
```

Comment:
interesting to see dependabot making these kind of changes, probably okay?

Reply:
Nope, this is from https://github.com/lensapp/lens/pull/3598 I don't know why it is part of the diff though...